### ISSUE TITLE
Online Documentation

### ISSUE THREAD

**Issue by Bergmaier** [Simmetrics/simmetrics#30]
Hi, this seems a very advanced library and in many aspects better than its competitors, but I cannot find any good documentation besides the readme - which links directly into the source of some java source files.
Is there any html javadoc website available?

**Comment by kinow** [comment]
Good idea @Bergmaier. Whenever I have to see how to use a metric, for now I just resort to [looking at the tests](https://github.com/Simmetrics/simmetrics/tree/59dc148f402da6a8a82ad8604a64fa35d1f70460/simmetrics-core/src/test/java/org/simmetrics).

**Comment by mpkorstanje** [comment]
@Bergmaier I'm afraid viewing the [example section](
https://github.com/Simmetrics/simmetrics/tree/master/simmetrics-example/src/main/java/org/simmetrics/example) on github is the best I can offer.

If any one wants to improve on it please feel free to send a PR.
